From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Wolfgang Bumiller <w.bumiller@proxmox.com>
Date: Tue, 24 Jul 2018 16:42:26 +0200
Subject: [PATCH] apparmor: allow start-container to change to lxc-**

For generated profiles with apparmor namespaces we get
profile names with slashes in them. To match those, we need
to allow changing to lxc-**, not just lxc-*.

Signed-off-by: Wolfgang Bumiller <w.bumiller@proxmox.com>
(cherry picked from commit 242a9fa7ee7e9f524de5a23917faa846ea525622)
---
 config/apparmor/abstractions/start-container | 1 +
 1 file changed, 1 insertion(+)

diff --git a/config/apparmor/abstractions/start-container b/config/apparmor/abstractions/start-container
index 414d058b..3df9883e 100644
--- a/config/apparmor/abstractions/start-container
+++ b/config/apparmor/abstractions/start-container
@@ -40,5 +40,6 @@
   pivot_root /usr/lib*/*/lxc/**,
 
   change_profile -> lxc-*,
+  change_profile -> lxc-**,
   change_profile -> unconfined,
   change_profile -> :lxc-*:unconfined,
-- 
2.11.0

